Makefile

```diff
@@ -178,6 +178,59 @@ build-gpu-cuda-118-base:
 		-t $(DOCKERHUB_REGISTRY)/$(GPU_CUDA_118_BASE_NAME)-$(VERSION) \
 		.
 
+NGC_PYTORCH_PREFIX := nvcr.io/nvidia/pytorch
+NGC_TENSORFLOW_PREFIX := nvcr.io/nvidia/tensorflow
+NGC_PYTORCH_VERSION := 23.10-py3
+NGC_TENSORFLOW_VERSION := 23.11-tf2-py3
+NGCPLUS_BASE_PYTORCH := ngc-pytorch
+NGCPLUS_BASE_DEEPSPEED := ngc-deepspeed
+NGCPLUS_BASE_TENSORFLOW := ngc-tensorflow
+
+.PHONY: build-ngc-pytorch
+build-ngc-pytorch:
+	docker build -f Dockerfile-ngc-pytorch \
+		--build-arg BASE_IMAGE="$(NGC_PYTORCH_PREFIX):$(NGC_PYTORCH_VERSION)" \
+		-t $(DOCKERHUB_REGISTRY)/$(NGCPLUS_BASE_PYTORCH)-$(NGC_PYTORCH_VERSION)-$(SHORT_GIT_HASH) \
+		-t $(DOCKERHUB_REGISTRY)/$(NGCPLUS_BASE_PYTORCH)-$(NGC_PYTORCH_VERSION)-$(VERSION) \
+		.
+
+.PHONY: build-ngc-deepspeed
+build-ngc-deepspeed:
+	docker build -f Dockerfile-ngc-pytorch \
+		--build-arg BASE_IMAGE="$(NGC_PYTORCH_PREFIX):$(NGC_PYTORCH_VERSION)" \
+		--build-arg DEEPSPEED_PIP="deepspeed==0.11.1" \
+		-t $(DOCKERHUB_REGISTRY)/$(NGCPLUS_BASE_DEEPSPEED)-$(NGC_PYTORCH_VERSION)-$(SHORT_GIT_HASH) \
+		-t $(DOCKERHUB_REGISTRY)/$(NGCPLUS_BASE_DEEPSPEED)-$(NGC_PYTORCH_VERSION)-$(VERSION) \
+		.
```

Comment:
out of curiosity, why do we need this to be separate from the ngc-pytorch image?

Reply:
We can have just 1. `ngc-pytorch` or `ngc-pytorch-deepspeed`?